package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "greensms",
-  "version": "1.0.2",
+  "version": "1.1.0",
   "description": "GREENSMS API: SMS, Viber, Voce, Call, HLR, Pay",
```

Comment:
GREENSMS API: SMS, Call, Voice, VK, WhatsApp, Viber, HLR, Pay

Reply:
Updated the content as well as the node version.